zerver/views/home.py

```diff
@@ -131,6 +131,16 @@ def compute_navbar_logo_url(page_params: Dict[str, Any]) -> str:
         navbar_logo_url = page_params["realm_logo_url"]
     return navbar_logo_url
 
+def compute_show_video_chat_button(realm: Realm, page_params: Dict[str, Any]) -> bool:
+    if realm.video_chat_provider == Realm.VIDEO_CHAT_PROVIDERS['disabled']['id']:
```

Comment:
Would we be better off just doing the frontend implementation, and doing a $("...").toggle(show_video_chat_button) on the compose box element inside `compose.initialize()` depending on the setting?  It makes me a little sad to duplicate this.

Reply:
Yes, it is being duplicated.
Will add toggle part in compose.initialize().